### ISSUE TITLE
OpenRCT Game Crash (after making custom scenario)

### ISSUE THREAD

**Issue by alasiamilan** [OpenRCT2/OpenRCT2#21650]
### Operating System

Windows 11 Home 64-bit

### OpenRCT2 build

OpenRCT2 v.0.4.9

### Base game

RollerCoaster Tycoon 2, RollerCoaster Tycoon Classic

### Describe the issue

Hi there! I really hope I am doing this correctly. I don't typically have any issues with OpenRCT2, so I don't often have to reach out for assistance.

I was creating a custom scenario in OpenRCT2. I completed every part of the set-up, and when I went to save my scenario, I pressed "save" on the dialogue window, and the game froze. Then I received this pop-up error message.

![Screenshot_7](https://github.com/OpenRCT2/OpenRCT2/assets/50000043/8db6e1b1-049e-450e-bbb4-8c849e462144)

I am pretty sure the custom scenario was saved (I located it in my saved games folder), but now I am not able to get back into OpenRCT2, as every time I attempt to start the game, I am met with that pop-up message.

Thank you in advance for any assistance!

### Steps to reproduce

1. Open the game (I receive an error message when I attempt to open the game).

### Attachments

[31e20a56-ba8c-4c54-8595-81a03a2747ec(a172405_x86-64).dmp.gz](https://github.com/OpenRCT2/OpenRCT2/files/14713779/31e20a56-ba8c-4c54-8595-81a03a2747ec.a172405_x86-64.dmp.gz)

**Comment by Gymnasiast** [comment]
Looking at the corresponding crash report, it seems the game is crashing while trying load the images for an object.

If the information in our crash tracker is current, you installed OpenRCT2 in `E:\OpenRCT2`, is that correct?

If so, please open a Command Prompt, and run the following command:
```
E:\OpenRCT2\openrct2.com --verbose
```
and copy and paste the output here. If the output is too long to paste here, you can use a service like pastebin.

**Comment by alasiamilan** [comment]
Yes, that is correct for the install location of my game.
Here is the output: https://pastebin.com/PU9SEz54

**Comment by Gymnasiast** [comment]
Right, it looks like you have a *lot* of obsolete files from old versions of OpenRCT2 in that install directory. When installing a new version, make sure you don’t just extract it over the old one.

You should empty out E:\OpenRCT2 and reinstall OpenRCT2. This should not touch your saves (which are on the C: drive in `Documents`), but making a backup, or moving the files instead of deleting them, is never a bad idea.

**Comment by janisozaur** [comment]
It should still not crash, ideally just inform the user their data is somehow corrupted
